### ISSUE TITLE
Custom address, cannot proceed?

### ISSUE THREAD

**Issue by Aquive** [trinos-nl/magento2-postcode-nl#14]
When I try to fill in a custom address by checking the checkbox and filling in the fields (street and city) and click the proceed button I get below validation error. I simply cannot proceed to the next step. Is this a known issue?

`"city" is required. Enter and try again.`.

**Comment by thijsdewitt** [comment]
Hi @Aquive Can you share more details on how to reproduce this? Also which version of this module are you using. And which version of the Hyva checkout are you using?

**Comment by Aquive** [comment]
Hi @thijsdewitt 

The project versions are:
- "trinos-nl/magento2-postcode-nl": "1.0.1"
- "hyva-themes/magento2-hyva-checkout": "1.1.10"

But I also tested with below versions with the same result:
- "trinos-nl/magento2-postcode-nl": "dev-main"
- "hyva-themes/magento2-hyva-checkout": "1.1.5"

I wanted to try to make a video to show whats going on. But There seems to be another issue now where the contents of both fields are just cleared after the request is done. The thing is, it doesn't seem stable somehow. Not sure what causes it.

See video.

Uploading Afrekenen.mp4…

**Comment by Aquive** [comment]
@thijsdewitt to be clear. This is not the issue I opened the bugreport for initially. I came acros this issue when I wanted to make a video to show the problem. That's when I encoutered this (no saving) problem.

There are no logs at all. When I look in the browser console I see 4 requests.

```
{"time":"2024-03-18T10:29:02+00:00", "remote_addr":"172.17.0.1", "remote_user":"", "host":"client.hypernode.local", "request":"POST /magewire/post/livewire/message/checkout.shipping-details.address-form HTTP/1.1", "status":"200", "body_bytes_sent":"640", "referer":"http://client.hypernode.local/checkout/", "user_agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36", "request_time":"0.465", "handler":"phpfpm", "country":"", "server_name":"client.hypernode.io", "port":"8080", "ssl_cipher":"", "ssl_protocol":""}
{"time":"2024-03-18T10:29:02+00:00", "remote_addr":"172.17.0.1", "remote_user":"", "host":"client.hypernode.local", "request":"POST /magewire/post/livewire/message/checkout.shipping-details.address-form HTTP/1.1", "status":"200", "body_bytes_sent":"640", "referer":"http://client.hypernode.local/checkout/", "user_agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36", "request_time":"0.466", "handler":"varnish", "country":"", "server_name":"client.hypernode.io", "port":"80", "ssl_cipher":"", "ssl_protocol":""}
{"time":"2024-03-18T10:29:03+00:00", "remote_addr":"172.17.0.1", "remote_user":"", "host":"client.hypernode.local", "request":"POST /magewire/post/livewire/message/checkout.shipping.methods HTTP/1.1", "status":"200", "body_bytes_sent":"378", "referer":"http://client.hypernode.local/checkout/", "user_agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36", "request_time":"0.246", "handler":"phpfpm", "country":"", "server_name":"client.hypernode.io", "port":"8080", "ssl_cipher":"", "ssl_protocol":""}
{"time":"2024-03-18T10:29:03+00:00", "remote_addr":"172.17.0.1", "remote_user":"", "host":"client.hypernode.local", "request":"POST /magewire/post/livewire/message/checkout.shipping.methods HTTP/1.1", "status":"200", "body_bytes_sent":"378", "referer":"http://client.hypernode.local/checkout/", "user_agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36", "request_time":"0.247", "handler":"varnish", "country":"", "server_name":"client.hypernode.io", "port":"80", "ssl_cipher":"", "ssl_protocol":""}

```

**Comment by thijsdewitt** [comment]
This is fixed in release 1.0.2
